setup.py

```diff
@@ -52,89 +52,94 @@ def get_varstring(fname="mdtools/version.py", varname="__version__"):
         author="Andreas Thum",
         author_email="andr.thum@gmail.com",
-        maintainer=get_varstring("mdtools/_metadata.py", "__maintainer__"),
-        maintainer_email=get_varstring("mdtools/_metadata.py", "__email__"),
+        maintainer=get_varstring(
+            "mdtools/_metadata.py", "__maintainer__"
+        ),
+        maintainer_email=get_varstring(
+            "mdtools/_metadata.py", "__email__"
+        ),
         url="https://github.com/andthum/mdtools",
         project_urls={
-            'Source': "https://github.com/andthum/mdtools",
-            'Documentation': "https://mdtools.readthedocs.io/en/latest/",
-            'Issue Tracker': "TODO",
-            'Feature Requests': "TODO",
+            "Source": "https://github.com/andthum/mdtools",
+            "Documentation": "https://mdtools.readthedocs.io/en/latest/",
+            "Issue Tracker": "TODO",
+            "Feature Requests": "TODO",
         },
         description="Python scripts to prepare and analyze molecular dynamics simulations",
         long_description=get_content("README.rst"),
-        long_description_content_type='text/x-rst',
+        long_description_content_type="text/x-rst",
         license=get_varstring("mdtools/_metadata.py", "__license__"),
         classifiers=[
-            "Programming Language :: Python :: 3 :: Only",
-            "Operating System :: Unix",
             "Development Status :: 3 - Alpha",
+            "Environment :: Console",
+            "Intended Audience :: Science/Research",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
             "Natural Language :: English",
-            "Intended Audience :: Science/Research",
+            "Operating System :: Unix",
+            "Programming Language :: Python :: 3 :: Only",
+            "Programming Language :: Python :: 3.6",
+            "Programming Language :: Python :: 3.7",
+            "Programming Language :: Python :: 3.8",
             "Topic :: Scientific/Engineering",
             "Topic :: Scientific/Engineering :: Chemistry",
             "Topic :: Scientific/Engineering :: Physics",
             "Topic :: Scientific/Engineering :: Bio-Informatics",
+            "Topic :: Software Development :: Libraries :: Python Modules",
             "Topic :: Utilities",
         ],
         keywords="molecular dynamics simulation analysis preparation chemistry physics materials science",
-        packages=setuptools.find_packages(include=['mdtools']),
+        packages=setuptools.find_packages(include=["mdtools"]),
         include_package_data=True,
-        python_requires=">=3.6, <=3.8",
-        install_requires=["psutil >=5.7, <6.0",
-                          "numpy >=1.18, <2.0",
-                          "scipy >=1.5, <2.0",
-                          "matplotlib >=3.2, <3.3",
-                          "MDAnalysis >=1.0, <2.0",
-                          "pyemma >=2.5, <3.0"]
+        python_requires=">=3.6",
```

Comment:
Why drop the upper bound? In `classifiers` only 3.6, 3.7, 3.8 are listed.

Reply:
You are right, it should be `<3.9`, because MDAnalysis requires Python <3.9. I must have messed something in the commit.